python/functional-programming/sum-all-map-functions/map-built-in-iii.md

```diff
@@ -0,0 +1,78 @@
+---
+author: lior-bd
+type: normal
+category: must-know
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+---
+
+# Lambda functions with map
+
+---
+
+## Content
+
+We can even use *lambda functions*[1] with `map`:
+
+```python
+scores = [1, 0, 4, 5]
+print(list(map(lambda x: x+2, scores)))
+# Result: [3, 2, 6, 7]
+```
+
+The above lambda function adds 2 to each number in our list.
+
+> Remember, in functional programming, functions can take functions. So, instead of a list of data, `map` may take a list of functions, and output a list of new functions. This gives a sense of how versatile and useful the `map` function can be.
```

Comment:
Maybe make this shorter and more focused on a specific point?

One idea:

```suggestion
> 💡 Remember, instead of a list of data, `map` may take a list of functions, and output a list of new functions. 
```